features/wishlist.feature

```diff
@@ -0,0 +1,20 @@
+Feature: The wishlist service back-end
+    As a e-Commerce Website
+    I need a RESTful wishlist service
+    So that I can keep track of all users' wishlists and their contents
+
+Scenario: Create a Wishlist
+    When I visit the "home page"
+    And I set the "Wishlist Name - Create" to "WISHLIST"
+    And I set the "Customer ID - Create" to "10"
+    And I press the "Create" button
+    Then I should see the message "Success"
+    When I copy the "Wishlist ID - Create" field
+    And I press the "Clear" button
+    Then the "Wishlist ID - Search" field should be empty
+    And the "Wishlist Name - Search" field should be empty
+    And the "Customer ID - Search" field should be empty
+    When I paste the "Wishlist ID - Search" field
+    And I press the "Search" button
+    Then I should see "1" wishlist(s)
```

Comment:
Also here for LIST APIs we may need a background and some dummy data, and if we add the background at the beginning there would be more than 1 wishlists

Reply:
If we search with a wishlist ID, we will only see 1 wishlist

Reply:
Yeah, I agree. I believe we need to work with the BDD format:
`Given` (set of preconditions)
`When` (something happens)
`Then` (testable outcome)

I feel the `Given` is missing here?

Reply:
You are right. Didn't notice that.